api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.4.x.xml

```diff
@@ -112,6 +112,25 @@
 								 referencedTableName="order_group"
 								 referencedColumnNames="order_group_id" />
     </changeSet>
+        
+    <changeSet id="20190816-Trunk-5652" author="gitacliff">
+		<preConditions onFail="MARK_RAN">
+			<not>
```

Comment:
Was this also in the ticket description?

Reply:
@dkayiwa  the Preconditions element can be attached to  a changeset to control the execution of an update based on the state of the database and in this case its housing the `<not>` element just like in the all other changesets adding new columns to the order_group table

Reply:
I mean the entire changeset.

Reply:
@dkayiwa i have made some changes ,,please review